pkg/prometheus/promcfg.go

```diff
@@ -924,7 +958,9 @@ func (cg *configGenerator) generateAlertmanagerConfig(version semver.Version, am
 		{Key: "scheme", Value: am.Scheme},
 	}
 
-	cfg = addTLStoYaml(cfg, am.TLSConfig)
+	// TODO: If we want to support secret refs for alertmanager config tls
```

Comment:
Did we open an issue for this?

Reply:
I will open an issue the moment this PR is merged.